docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -822,6 +814,18 @@ Every task has some expansions available by default:
 -   `${__project_aws_ssh_key_value}` is the unencrypted PEM encoded PKCS#1 private key 
     returned along with `${__project_aws_ssh_key_name}`.
 
+The following expansions are available unless a task was from an
+[untracked branch](Repo-Level-Settings.md#how-to-use-pr-testing-for-untracked-branches).
+
+-   `${project}` is the project identifier the task belongs to
+-   `${project_identifier}` is the project identifier the task belongs
+    to // we will be deprecating this, please use `${project}`
+-   `${project_id}` is the project ID the task belongs to (note that for
+    later projects, this is the unique hash, whereas for earlier
+    projects this is the same as `${project}`. If you aren't sure which
+    you are, you can check using this API route:
+    `https://evergreen.mongodb.com/rest/v2/<project_id>`)
```

Comment:
nit: I know it was already like this, but would be preferable to link this to the route's docs.

Reply:
Linked it!